plugins/inputs/smart/smart.go

```diff
@@ -865,7 +865,6 @@ func (m *Smart) gatherDisk(acc telegraf.Accumulator, device string, wg *sync.Wai
 					}
 
 					if err := parse(fields, deviceFields, matches[2]); err != nil {
-						acc.AddError(fmt.Errorf("error parsing %s: %q: %w", attr.Name, matches[2], err))
```

Comment:
I'm not sure I can follow. `acc.AddError()` will _not_ cancel further processing but logs the error and increases the error-counter (for the internal metric) IMO. So where do you see a difference to the previous behavior despite from now silently ignoring the error?

Reply:
This aligns the behavior with the other parsing in the plugin. 

The smart plugin for better or worse heavily relies on the use of regex throughout. We currently do not complain about other failures or parsing issues, except here. The user cannot fix the issue when this error comes up, so rather than complaining and making the user think they need to change something, remove the error message.